### PR TITLE
data:AatifSohel

### PR DESCRIPTION
**Pull request by aatifsohel**

### CODE REVIEW

**Review by Tejender1521 — APPROVED** [review]
Everything looks good to me

File: data/aatifsohel.json [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/aatifsohel/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/aatifsohel/",
```
Comment (Njong392) [inline data/aatifsohel.json:25]: This instagram link appears to lead to a 404 page. Maybe there is some typo you made?

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you so much for contributing to LinkFree 🎉 Please take a look at the suggested changes I have made.

File: data/aatifsohel.json (hunk 1 above)
Comment (aatifsohel) [reply data/aatifsohel.json:25]: NO! I temporarily deactivated instagram account. That's why it is showing 404 page.

**Review by Njong392 — APPROVED** [review]
Looks great!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**